packages/typespec-go/.scripts/tspcompile.js

```diff
@@ -21,11 +21,11 @@ const cadlRanch = {
   'extensiblegroup': ['type/enum/extensible'],
   'fixedgroup': ['type/enum/fixed'],
   'emptygroup': ['type/model/empty'],
-  //'enumdiscriminatorgroup': ['type/model/inheritance/enum-discriminator'],
-  //'nesteddiscriminatorgroup': ['type/model/inheritance/nested-discriminator'],
-  //'nodiscriminatedgroup': ['type/model/inheritance/not-discriminated'],
-  //'recursivegroup': ['type/model/inheritance/recursive'],
-  //'singlediscriminatorgroup': ['type/model/inheritance/single-discriminator'],
+  'enumdiscgroup': ['type/model/inheritance/enum-discriminator'],
+  //'nesteddiscgroup': ['type/model/inheritance/nested-discriminator'],
```

Comment:
This scenario is multiple discriminator properties on a type. The TS team is debating on the validity/usefulness of this and might remove it. So we're skipping this for the time being.